site/_resources/upgrading.md

```diff
@@ -10,17 +10,80 @@ This document describes the changes needed to upgrade your Contour installation.
 <div class="alert-deprecation">
 <b>Deprecation Notices</b><br>
 <ul>
-<li>Contour no longer supports TLS 1.1.</li>
+<li>The contour_httpproxy_total Gauge metric has been removed in favor of the similarly named contour_httpproxy Gauge metric.</li>
```

Comment:
This section is mainly the only one I actually changed, besides some cleanup later in this file